docs/mllib-statistics.md

```diff
@@ -422,6 +422,41 @@ for i, result in enumerate(featureTestResults):
 
 </div>
 
+Additionally, MLlib provides a 1-sample, 2-sided implementation of the Kolmogorov-Smirnov test
```

Comment:
`Kolmogorov-Smirnov` -> `Kolmogorov-Smirnov (KS)`

Otherwise, we use `KS` without definition.